components/operators/openshift-ai/instance/components/model-server-pod-sizes/patch-rhoai-dashboard.yaml

```diff
@@ -12,6 +12,14 @@ spec:
         requests:
           cpu: '1'
           memory: 4Gi
+    - name: Small-Medium
```

Comment:
This style of resource configuration is depreciated with accelerator profiles.

Consider using hardware profiles instead:
https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.22/html/working_with_accelerators/working-with-hardware-profiles_accelerators#creating-a-hardware-profile_accelerators